CODE_OF_CONDUCT.md

```
@@ -0,0 +1,90 @@
+### Our Pledge
+
+We, as contributors, maintainers, and participants of AstraGuard AI, pledge to make participation in this project a professional, respectful, and inclusive experience for everyone.
+
+AstraGuard AI is a mission-critical, security-focused open-source system. Collaboration here requires not only technical skill, but responsibility, integrity, and respect for others.
+
+We are committed to fostering an environment that respects the dignity, rights, and contributions of all individuals, regardless of race, ethnicity, caste, color, age, physical characteristics, neurodiversity, disability, sex or gender, gender identity or expression, sexual orientation, language, philosophy or religion, national or social origin, socio-economic position, level of education, or other personal characteristics.
+
+---
+### Expected Behavior
+
+   - All participants are expected to:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)